en/docs/fragments/register-an-identity-provider.md

```diff
@@ -1,6 +1,6 @@
 ## Register an identity provider
 
-1. Log in to the Management Console (`https://<IS_HOST>:<PORT>/carbon`) using admin/admin credentials. 
+1. Log in to the [Management Console](`https://<IS_HOST>:<PORT>/carbon`) using admin/admin credentials. 
```

Comment:
will having  Management Console enclosed in`[ ]` make it a link?

Reply:
yes, modified now.